dokka-subprojects/plugin-base/src/main/resources/dokka/styles/style.css

```
@@ -580,6 +580,10 @@ td:first-child {
     border-bottom: 2px solid var(--active-tab-border-color);
 }
 
+.section-tab:focus, .section-tab:active {
```

`focus-visible` would be a slightly better option here because it only cause outline when navigation via keyboard and not when clicking which leads to a slightly clear UI

```suggestion
.section-tab:focus-visible {
```